File: Exercise6.py
#Exercise 6: Retirement Calculator

import datetime
year = datetime.date.today().year
age = int(input("What is your current age? "))
retire_age = int(input("At what age would you like to retire? "))
time_left = retire_age - age
print(f"You have {time_left} left until you can retire.")
print(f"It's {year}, so you can retire in {year+time_left}.")

File: Exercise6.rb
require 'date'
year = Date.today.year.to_i

print "What is your current age? "
age = gets.to_i
print "At what age would you like to retire? "
retire_age = gets.to_i 

time_left = retire_age - age
retire_year = year + time_left
puts "You have #{time_left} years left until you can retire."
puts "It's #{year}, so you can retire in #{retire_year.to_s}."
